config-dev.json

```diff
@@ -146,7 +146,7 @@
       "nativeAsset": "BASEETH",
       "networkTicker": "BASEETH",
       "nodeUrls": [
-        "https://mainnet.base.org"
+        "https://api.blockchain.info/base/nodes/rpc/"
```

Comment:
Don't we want: `https://api.dev.blockchain.info/base/nodes/rpc` here? 🤔 
It does not change much but I think it's exposed as well

Reply:
Uhh I'm not sure. The prod one definitely works, but I'm getting a 503 from staging.

```
curl -X POST https://api.staging.blockchain.info/base/nodes/rpc/  -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_blockNumber","params":[],"id":1}'

<html>
<head><title>503 Service Temporarily Unavailable</title></head>
<body>
<center><h1>503 Service Temporarily Unavailable</h1></center>
<hr><center>nginx</center>
</body>
</html>
```

Reply:
But dev does work: 

```
curl -X POST https://api.dev.blockchain.info/base/nodes/rpc  -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_blockNumber","params":[],"id":1}' 
{"jsonrpc":"2.0","id":1,"result":"0x12ae5d7"}
```

Reply:
```suggestion
        "https://api.dev.blockchain.info/base/nodes/rpc/"
```